diofant/utilities/codegen.py

```diff
@@ -1369,7 +1369,8 @@ def _call_printer(self, routine):
                 raise CodeGenError("unexpected object in Routine results")
 
             constants, not_supported, oct_expr = octave_code(result.expr,
-                                                             assign_to=assign_to, human=False)
+                                                             assign_to=assign_to,
+                                                             human=False, inline=False)
```

Comment:
@cbm755, probably that was a bug in OctaveCodeGen, which is still valid for SymPy.  With inline=True - constants are never generated.

Reply:
Thank you!  I'll try to get around to fixing this.